cmake/thirdparty/BLTSetupHIP.cmake

```diff
@@ -41,32 +47,7 @@ message(STATUS "ROCM path:        ${ROCM_PATH}")
 message(STATUS "HIP version:      ${hip_VERSION}")
 
 # AMDGPU_TARGETS should be defined in the hip-config.cmake that gets "included" via find_package(hip)
-# This file is also what hardcodes the --offload-arch flags we're removing here
 if(DEFINED AMDGPU_TARGETS)
```

Comment:
It seems like there's no longer a point for having AMDGPU_TARGETS.

Reply:
Or is it used internally by hip?

Reply:
I don't see it in CMake's source, maybe it is in the FindRocm source. I would have expected this to only be used as a backwards compatiblity and maybe just set the CMake variable.

Reply:
I could do backwards compatibility but I think removal might be best since we are going to be having breaking BLT changes soon